Corona-Warn-App/src/main/res/values-de/dcc_ticketing_strings.xml

```diff
@@ -1,6 +1,14 @@
 <?xml version="1.0" encoding="utf-8"?>
 <resources xmlns:tools="http://schemas.android.com/tools" tools:ignore="MissingTranslation">
 
+    <!-- Error Handling -->
+    <!-- XTXT: Service Provider (without provider name) error message-->
+    <string name="dcc_ticketing_error_service_provider_error_no_name">Es trat ein Fehler bei der Verbindung mit dem Anbieter. Bitte kontaktieren Sie den Anbieter wenn das Problem weiterhin besteht.</string>
```

Comment:
Instead of "Es trat ein Fehler bei der Verbindung mit dem Anbieter." it should be "Es trat ein Fehler bei der Verbindung mit dem Anbieter auf." ("auf" at the end is missing). Please correct.